README.md

```diff
@@ -448,6 +468,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Web Frameworks
  * [amber](https://github.com/amberframework/amber) - Open source efficient and cohesive web application framework
+ * [HTTP::Server](https://crystal-lang.org/api/HTTP/Server.html) - HTTP server (Crystal stdlib)
```

Comment:
Maybe ` HTTP::Server` fits better in ***HTTP*** section.

Reply:
Maybe we should rather rename these sections to `HTTP client` and `HTTP server`.

Reply:
Good idea! But a question comes to my mind: can web frameworks be called HTTP servers? They're usually built on top of `HTTP::Server`.

Reply:
I don't think this matters, really.

Reply:
Please move it to `HTTP` section.